Initialtask/src/Repository/FileParserRepository.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\FileParser;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use League\Csv\Reader;
+
+class FileParserRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, FileParser::class);
+    }
+
+    public function loadData(string $filePath)
+    {
+        if (file_exists($filePath)) {
```

Comment:
guess, that current repository should load data from database only and do not need to know anything regarding file
Probably we don't need it due to comments to FileParser